### DISCUSSION TITLE
Laptop battery status?

### DISCUSSION THREAD

**Discussion by lkroeker** [dougiteixeira/proxmoxve#214] [Feature request] [answered]
Hi!

I've got the custom integration set up, and it's neat to have all the info in HA, but I'm missing battery info (because it's probably not a standard setup). My Proxmox server is running on an old laptop, so it has a built-in battery, and I'm trying to find a good way to have the info sent to HA for battery conditioning (keeping the battery between 30-80% charge). 

There are a number of parameters stored in the sys folder for the device, but the main one is capacity:
/sys/devices/LNXSYSTM:00/LNXSYBUS:00/PNP0A08:00/device:18/PNP0C09:00/PNP0C0A:00/power_supply/BAT0
(/sys/class/power_supply/BAT0/)
capacity (0-100)
or
uevent (all bat0 stats are seemingly stored here)

I'm wondering if there's an easy way to add this to my config manually?

**Comment by dougiteixeira** [comment, marked as answer]
I don't see the possibility of this being included in the API, if it exists it could be included in the integration.

However, you can use an alternative similar to [the script for sending CPU temperature](https://github.com/dougiteixeira/proxmoxve/blob/main/scripts_cron/cpu_temperature.md) information to send this information. Good luck!

**Reply by lkroeker** [reply]
Ah, I see.  Thanks for the reply!

For now I set up a Python script that runs on the server and sends it to HA via mqtt: https://community.home-assistant.io/t/using-proxmox-on-a-laptop-with-a-battery/584549

Cheers!
